### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/producers.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4878]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4878|
|Type|enhancement|
|Created|2011-02-11 05:45:33Z|
|Branch|https://github.com/twisted/twisted/tree/producers-sphinx-4878|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the producers howto

<details><summary>Searchable metadata</summary>

```
trac-id__4878 4878
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_producers_sphinx_4878 branches/producers-sphinx-4878
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297403133000000 1297403133000000
changetime__1300072013000000 1300072013000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30689]) Branching to 'producers-sphinx-4878'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30690]) Merge the producers parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4878](https://github.com/twisted/twisted/issues/4878)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_6"></a>|@lvh set status to `closed`|
|-|-|

(In [30996]) Merge producers.xhtml changes from producers-sphinx-4878.


Author: screwtape, jdb
Reviewer: lvh
Fixes: [#4878](https://github.com/twisted/twisted/issues/4878)
  
Fixed some links to API docs in producers.xhtml.
